components/Dialogs/examples/AlertComparison.swift

```diff
@@ -0,0 +1,142 @@
+/*
+ Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+import Foundation
+import MaterialComponents
+
+
+/// This interface allows a user to present a UIKit Alert Controller and a Material Alert
+/// Controller.
+class DialogsAlertComparison: UIViewController {
+
+  let materialButton = MDCFlatButton()
+  let UIKitButton = MDCFlatButton()
```

Comment:
`uikitButton` to avoid confusion with being a class name.

Reply:
Done.